apps/calendar/jsdoc.json

```diff
@@ -5,11 +5,7 @@
   },
   "source": {
     "include": ["apps/calendar/README.md", "apps/calendar/js"],
-    "exclude": ["apps/calendar/js/ext",
```

Comment:
ok i will make the change, can i leave the files edited in js/ext ? If they will be erased.